.gitattributes

```diff
@@ -0,0 +1,35 @@
+# Define the line ending behavior of the different file extensions
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text text=auto eol=lf
+
+*.php diff=php
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
+*.gif binary
+*.jpeg binary
+*.zip binary
+*.phar binary
+*.ttf binary
+*.woff binary
+*.woff2 binary
+*.eot binary
+*.ico binary
+*.mo binary
+*.pdf binary
+*.xsd binary
+*.exe binary
```

Comment:
Noticed we have more files in suite-nonsplit/registry-service/etc:
*.xcf binary
*.eps binary
*.psd binary
*.dll binary
*.doc binary
*.docx binary

Just information think about it

Reply:
I have used standard list of files after creation via spryk

Reply:
Those are only relevant as per repo
So if none of those are used here, we are good.